i18n/en.json

```diff
@@ -127,6 +133,9 @@
 	"metadata-start-date": "Start date:",
 	"metadata-wikis-label": "{{PLURAL:$1|Wiki|Wikis}}:",
 	"not-all-wikis-have-cats": "Categories not supplied for some wikis",
+	"num-abbrev-billion": "b",
+	"num-abbrev-million": "m",
```

Comment:
Wowee, brain farts I guess? As a software engineer obviously I would know about MB and GB! I guess it's just the context that threw me off.

The designs do have million with a lowercase m, though https://prtksxna.github.io/wmf-prototype-gm/event-filter-complete-nogreen.html To be frank I'm not sure

Reply:
In America, million is commonly abbreviated with just an m. The same is true for billion with b. But, I think this akin to slang and not real abbreviation. You wouldn't see it in a newspaper or magazine.

Reply:
Okay so it's not just me, then :) I guess we should try the metric system and see what people think. I did mark translations for the abbreviations as optional, which would only apply if we use an accepted universal format.

We could also change it to `m` and `b` in en-us.json so that it would only affect American English.